sirepo/job_supervisor.py

```diff
@@ -912,8 +912,7 @@ async def _send_op(op, compute_job_serial, prev_db):
 
         op.task = asyncio.current_task()
         op.pkdel("run_callback")
-        if not await _send_op(op, compute_job_serial, prev_db):
```

Comment:
I think we need to catch the top level await on a task. It's not clear to me that this will work as written.

Reply:
2f7887c01